.github/workflows/pokeone-api.yaml

```diff
@@ -1,12 +1,34 @@
 name: PokeOneApi
 on: [push]
 jobs:
-  Build-PokeOneApi:
+  build:
+    name: Build-PokeOneApi
     runs-on: ubuntu-latest
     steps:
+    - name: Set up JDK 11
```

Comment:
Für was braucht es denn Java/JDK? (genuinely curious)

Reply:
Müsste für Sonar-Scanner sein, weil dieser glaubs mit Java entwickelt wurde.

Reply:
I never ran the pipeline without this step. But the official template for the analysis of .Net projects in SonarCloud has this step explicit in it.